scripts/tests.bat

```diff
@@ -48,13 +48,15 @@ if "%TESTS%"=="On" (
     testeth.exe --verbosity 2
     echo Testing EVMJIT
     testeth.exe -t VMTests,StateTests --vm jit --verbosity 2
+    IF errorlevel 1 GOTO ON-ERROR-CONDITION
```

Comment:
Batch sucks, but at least this gives us error-handling.
Next step would be nuke-and-pave with PowerShell.